test/e2e/policies.test-suite/balloons/n4c16/test10-health-checking/code.var.sh

```diff
@@ -8,22 +8,18 @@ terminate nri-resmgr
 nri_resmgr_config=fallback launch nri-resmgr
 vm-command "kubectl apply -f nri-resmgr-configmap.yaml"
 
-# Give enough time for the Kubernetes to consider the container not ready.
-sleep 10
-
 # Check if nri-resmgr readiness probe failed as expected
-if ! vm-command "kubectl get event -n kube-system --field-selector involvedObject.name=$POD | grep -q 'Readiness probe failed'" 2>&1; then
-    error "Expected readiness probe to fail, but got it succeeded"
+if ! vm-command "kubectl -n kube-system wait --for=condition=Ready=false pods/$POD --timeout=60s"; then
```

Comment:
Can't we use here the function from vm?

Reply:
@jukkar my bad, I again missed the balloons policy :) Fixed it now. Thanks for the catch.